scripts/variantstore/wdl/extract/run_avro_query_for_sample_info.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+import math
+import argparse
+
+from google.cloud import bigquery
+from google.cloud.bigquery.job import QueryJobConfig
+
+import run_avro_query
+import utils
+
+
+def get_number_of_partitions(dataset_name, project_id):
+    query_labels_map = {
+        "id": "construct_sample_info_avro_queries",
+        "gvs_tool_name": "gvs_extract_avro_files_for_hail",
+        "service": "gvs",
+        "team": "variants",
+        "managedby": "gvs_extract_avro_files_for_hail"
+    }
+
+    # Default QueryJobConfig will be merged into job configs passed in
+    # but if a specific default config is being updated (eg labels), new config must be added
+    # to the client._default_query_job_config that already exists
+    default_config = QueryJobConfig(labels=query_labels_map, priority="INTERACTIVE", use_query_cache=True)
+    client = bigquery.Client(project=project_id,
+                             default_query_job_config=default_config)
+
+    sql = f"SELECT max(sample_id) / 4000 as max_table_num FROM `{project_id}.{dataset_name}.sample_info`"
```

Comment:
surprising (to me at least) that in BQ an int divided by an int yields a float, but apparently it does...

Reply:
depends on how one does division.

`SELECT 8576/400`
gives 21.44, whereas

`SELECT div(8576,400)`
gives 21

It sounds like this code maybe could benefit from just using div() directly, but we'd have to handle the edge case where max(sample_id) / 4000 leaves no remainder instead of just adding 1 to the integer division result.  In some ways, this implementation might be simpler because it handles that case just fine.